### DISCUSSION TITLE
Page snapshot Error 5000: entity too large

### DISCUSSION THREAD

**Discussion by msoheib** [coddingtonbear/obsidian-web#120] [General]
Can I increase the max size of the payloads?

I cannot snapshot certain web pages on the longer side.

A POST request for an entire page snapshot gives me - "Could not send content to Obsidian: (Error Code 50000) Internal Server Error request entity too large"

Any way to increase the max size?

**Comment by coddingtonbear** [comment]
Maybe!   That's a limitation being returned to you from the underlying api, and there's a discussion here: https://github.com/coddingtonbear/obsidian-local-rest-api/discussions/73

**Comment by coddingtonbear** [comment]
This was fixed in https://github.com/coddingtonbear/obsidian-local-rest-api/discussions/73 recently!
